Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs

```diff
@@ -107,7 +107,7 @@ private void Detach()
                 foreach (var device in _devices)
                 {
                     // For now, we only handle devices that can take vibration messages.
-                    if (!device.Messages.Contains(typeof(SingleMotorVibrateCmd).Name))
+                    if (!device.Messages.Keys.Contains(typeof(SingleMotorVibrateCmd).Name))

```

Comment:
We should just make a Device.SupportsMessage function to shorten this a bit. Totally doable in a followup though.